streams-designer/README.md

```diff
@@ -0,0 +1,155 @@
+In this tutorial you will:
+
+1. Set up IBM services Cloudant, Streaming Analytics, Watson Studio, and Db2 Warehouse on Cloud.
+1. Create a Python code operator that will load Cloudant documents from the `_changes` endpoint.
+1. Edit the output schema for the Python code operator.
+1. Create a Db2 operator and add the connection details.
+1. Map the Code node schema with the target columns in the Db2 table.
+1. Save and run the streams flow.
+
+**N.B. This is an example only intended as a _starting point_ for using IBM Streams Designer to extract data from Cloudant and insert into Db2 Warehouse on Cloud. It is _not_ intended as a drop-in replacement for the deprecated Cloudant warehouse integration. In particular this example does not create or alter Db2 tables or handle document deletion.**
+
+## Before you begin 
+
+These are the IBM Bluemix services required to run this tutorial:
+1. [Cloudant](https://console.bluemix.net/catalog/services/cloudant-nosql-db)
+1. [Db2 Warehouse on Cloud](https://console.bluemix.net/catalog/services/dashdb)
+1. [Streams Designer](https://www.ibm.com/cloud/streams-designer)
+
+# Setting up IBM Cloud services
+
+## Cloudant
+1. [Create an IBM Cloudant](../cloudant/create.md) instance.
+1. [Replicate the tutorial data](../cloudant/replicate.md).
+
+## Streaming Analytics
+1. [Create an IBM Streaming Analytics](../streaming-analytics/create.md) instance.
+
+## Watson Studio
+1. [Creating an IBM Watson Studio](../watson-studio/create.md) instance.
+
+## Db2 Warehouse on Cloud
+1. [Create an IBM Db2 Warehouse on Cloud](../db2/create.md) instance.
+1. [Create a Db2 table to house the data](../db2/animaldb_table.md).
+
+## Create a Watson Studio project
+1. In the Watson Studio instance, open the Watson Studio app by pressing `Get Started`.
+![](get-started-watson-studio.png)
+1. Create a new project by selecting `New Project`.
+![](new-project-watson-studio.png)
+1. Select the `Streams Designer` tile and press OK.
+1. On the **New project** screen, add a name and optional description for the project, then press **Create**.<br/>
+**Note**: The previously created Streaming Analytics service should be pre-selected as seen in the screenshot.  If the 
+Cloud Object Storage instance was not created beforehand, you will have the option of adding one.
+![](create-new-project-watson-studio.png) 
+
+## Create a Streams flow
+1. Create a new Streams flow under the `Add to project` drop-down list.
+![](streams-flow-watson-studio.png)
+1. On the **New Streams Flow** screen, add a name and optional description for the project. 
+The Streaming Analytics service drop-down box should already be filled in.  
+Select the **Manually** box, then press **Create**.
+![](create-streams-flow-watson-studio.png)
+
+## Create operators in the Streams Designer editor
+The first node we'll create is a Python node for loading documents from Cloudant.
+1. Drag and drop the Code operator under **Sources** on to the canvas.
```

Comment:
markup "Code" same as above and apply throughout

Reply:
Fixed in 38e8498.